imaginairy/enhancers/describe_video.py

```diff
@@ -0,0 +1,278 @@
+import json
+import os
+import shutil
+
+import cv2
+import numpy as np
+import openai
+from skimage import metrics
+
+from imaginairy import LazyLoadingImage
+from imaginairy.enhancers.describe_image_blip import generate_caption
+
+# openai.api_key = "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"
```

Comment:
I know...I disabled the key and am trying to push an update but for some reason it's taking forever with no error message.

Reply:
!